content/ReleaseNotes/TrueCommand/2.0-Beta.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "2.0-Beta"
+weight: 10
+---
+
+**May 4, 2021**
+
+[TrueCommand 2.0](https://www.truenas.com/docs/truecommand/devnotes/) is now available in BETA version and extends TrueCommand to manage, create, and delete datasets and snapshots. TrueNAS SCALE clusters and their datasets can also be created and managed.  Many other improvements significantly simplify storage operations and dramatically improve performance and responsiveness. TrueCommand 2.0 is expected to be released in June.
+
+2.0 greatly improves the predictability, reliability, and total cost of ownership of production systems. Significant new features include:
+
+* Faster Database
+* Real-Time Stats
+* Storage Navigator
+* iSCSI Manager
+* Improved System-Down alerts
+* Secure Web-Proxy for Cloud
+* TrueNAS SCALE cluster UI
+
+
+[TrueCommand](https://www.truenas.com/truecommand/) was launched in 2019 as a single pane of glass management system for TrueNAS fleets. [TrueCommand 1.3]({{< relref "/ReleaseNotes/TrueCommand/1.3.md" >}}) has been adopted by well over three thousand organizations to manage their NAS fleets. TrueCommand is deployed as a Container, as a VM, or as the [TrueCommand Cloud](https://www.ixsystems.com/blog/truenas-with-truecommand-cloud/) service. TrueCommand is free for users with less than 50 drives and is very affordable for larger organizations, where TrueCommand coordinates a team of people to manage many systems on a 24 x 365 basis.
+
+## 2.0-Beta Changelog
+
+### New Feature
```

Comment:
Do we want to list any of the large epic tickets here for Cluster management, iSCSI manager or other big features?

Reply:
this is a straight port of the "New Feature" type tickets from https://jira.ixsystems.com/secure/ReleaseNote.jspa?projectId=10300&version=12906 - do you want the Epic section combined in here or done as a separate section?